### PR TITLE
add bash start script

### PR DESCRIPTION
**Pull request by rixombea**

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+rpc="test"
+read -p 'Enter Wallet Name: ' user
+read -p 'Enter Wallet Pass: ' pass
+
+if [[ -e "scripts/${user}" ]];then
+  echo "file exists"
+  ./scripts/turtle-service.exe -w ./scripts/${user} -p "${pass}" --rpc-password test --daemon-address public.turtlenode.io --enable-cors '*'
```
Comment (Watt3r) [inline start.sh]: This does not work as you are calling ./turtle-service.exe, add the linux binary from the official turtlecoin repo and change the two calls

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  xdg-open ./scripts/index.html
+else
+  echo "file doesn't exist"
+  ./scripts/turtle-service.exe -g -w ./scripts/${user} -p "${pass}" --rpc-password test --daemon-address public.turtlenode.io --enable-cors '*'
```
Comment (Watt3r) [inline start.sh]: This does not work as you are calling ./turtle-service.exe, add the linux binary from the official turtlecoin repo and change the two calls

Comment (rixombea) [reply start.sh]: binary name updated, was only saw the 'exe' binary on the repo.

should I add the pre-compiled turtle-service binary into the repo?

**Comment by brandonlehmann** [comment]
See #17

**Comment by Watt3r** [comment]
What is your name on discord? @brandonlehmann gave you the bounty reward of 20k Turtle

**Comment by rixombea** [comment]
@labaylabay

**Pull request closed without merging**